### ISSUE TITLE
Allow `:focus-visible` pseudo-selector to be set in `theme.json`

### ISSUE THREAD

**Issue by huubl** [WordPress/gutenberg#68520]
## What problem does this address?

Currently `:focus-visible` cannot be set via `theme.json` because the [VALID_ELEMENT_PSEUDO_SELECTORS](https://github.com/WordPress/gutenberg/blob/trunk/lib/class-wp-theme-json-gutenberg.php#L577-L580) list does not include :focus-visible.

## What is your proposed solution?

Add `:focus-visible` to the `VALID_ELEMENT_PSEUDO_SELECTORS` list to allow the use of the `:focus-visible` pseudo-selector in `theme.json`, providing more flexibility.

**Comment by huubl** [comment]
Created a pull request:
https://github.com/WordPress/gutenberg/pull/68521

**Comment by Bovelett** [comment]
@afercia Wouldn't this also qualify for the accessibility tag? I don't think this is "just" an enhancement.
